server_config.go

```diff
@@ -100,6 +101,10 @@ func (config *ServerConfig) Parse(configMap map[interface{}]interface{}, pathCon
 				if err != nil {
 					return fmt.Errorf("error loading identity: %v", err)
 				}
+
+				if err := config.DefaultIdentity.WatchFiles(); err != nil {
```

Comment:
Without knowing the identity library at all, will this apply to both the root identity block and to any/all blocks declared throughout the rest of the config? Can routers be updated too?

Reply:
This PR is xweb only, so its scope is only xweb. xweb supports whatever identity is passed in as the default plus all sub-identity sections, which are all marked as watching files.

The changes for the router and controller side are in the ziti repo and can not be in this PR.